app.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'lib/takeaway'
+require 'sinatra/base'
+require 'twilio-ruby'
+
+class App < Sinatra::Base
+  def takeaway(takeaway_class = TakeAway)
+    takeaway_class.new
+  end
+
+  def responder(responder_class = Twilio::TwiML::MessagingResponse)
+    responder_class
+  end
+
+  get '/' do
+    takeaway.print_menu.gsub("\n", "<br>")
+  end
+
+  post '/sms' do
```

Comment:
I'd look at refactoring this action to keep it down to the slimmest possible.

- what is the point of it? keep only what is essential and hide away params fetching and perhaps even thinking of a different way to instantiate the responder.